build-system/tasks/extension-helpers.js

```diff
@@ -430,14 +433,21 @@ async function buildExtension(
   if (options.compileOnlyCss && !hasCss) {
     return;
   }
-  const path = 'extensions/' + name + '/' + version;
+  const extensionPath = 'extensions/' + name + '/' + version;
```

Comment:
I've renamed this to `extDir` in #33298 so that everything fits on one line. Mind merging this change with that?